### ISSUE TITLE
dev configuration rebuilds 30 times on running start:dev

### ISSUE THREAD

**Issue by sgordon3** [ezavile/postcss-typescript-css-example#2]
Although the dev config runs and eventually builds, building stylesheets with tons of rules could cause the build process to be rather lengthy. Have you seen this? The problem occurs as a result of trying to build the ts files and also import them in the same build process. I figured that since the files are written synchronously that it would block the next process from running.

Steps to reproduce:
1. set the devServer quiet option to false
2. temporarily remove the FriendlyErrorsWebpackPlugin
3. run npm start:dev

See this:
webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

webpack: Compiled successfully.
webpack: Compiling...

note: removing the imports for the titleStyle and cardStyle files prevents this issue.

**Comment by ezavile** [comment]
Yes, I've seen this problem, and I tried to change the way to write the TS
files using an async method, but the result it's the same. I'll try to fix
this detail, thanks!

2017-09-25 11:39 GMT-05:00 Stewart E Gordon III <notifications@github.com>:

**Comment by sgordon3** [comment]
I thought that webpack dynamic imports using import().then might also resolve, but it made no difference. If I find a solution, I'll submit a pull request.

**Comment by ezavile** [comment]
That would be great 👍 Thanks!

2017-09-25 20:00 GMT-05:00 Stewart E Gordon III <notifications@github.com>:

**Comment by sgordon3** [comment]
Its also possible this is a webpack bug.
